src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterMen = people.filter(({ sex, died }) => (century)
```

Comment:
Only name of functions should start with verbs, but it is not a function. This variables store array with men.
```suggestion
  const filteredMen = people.filter(({ sex, died }) => (century)
```